rebar.config

```diff
@@ -14,3 +14,4 @@
 {cover_enabled, true}.
 {cover_opts, [verbose]}.
 {cover_export_enabled, true}.
+%{ct_readable, false}.
```

Comment:
Why add commented-out stuff?

Reply:
I found myself turning this option on and off pretty frequently.